Visual Studio compatibility

fieldOfView opened the pull request:
This patch has two minor fixes for compatibility with Visual Studio, and adds project files for the examples. The project files expect python 2.7 installed in C:\Python27, but if you have Python 2 installed in another directory you can set an environment variable named PYTHON pointing to the proper directory.

fieldOfView committed:
Add project files and minor fixes for Visual Studio

fieldOfView committed:
Allowed for setting python install with the PYTHON environment variable
A default path is provided if the variable does not exist

chaosct commented:
Awesome!
Would you care to add the information about the PYTHON environment variable to the readme?
Then I'll merge it.

fieldOfView committed:
Update README.md

fieldOfView commented:
Updated the readme with instructions
I have also added a small note about installing the python-dev package under Linux. I hope you don't mind me shoving that in with this pr.

chaosct commented:
Nice.
